pandunia-loge.csv

```diff
@@ -1797,7 +1804,7 @@ labe|||fas:urd:(lab), hin:लब (lab), tgl:labi, spa:labio, por:lábio|lip||lab
 lada|||eng:load, deu:laden, pol:ładować|load (to charge)||cargar||||||||||||||ladata|naładować, załadować, obładować, ładować
 lage|||eng:lake, spa:por:ita:lago, fra:lac|lake|lac|lago|lago|||湖|湖||||||||||jezioro
 Lagos|cite|NG||Lagos||||||||||||||||Lagos|
-laje|||zho:垃圾 (lājī)|garbage (rubbish, trash)||basura (desechos)|||||||||||||rubo|roska|śmieć, odpadek
+laje|||zho:垃圾 (lājī)|garbage (rubbish, trash)|ordures|basura (desechos)|refugo (escória)|мусор (дрянь)||废物 (垃圾)|ゴミ (廃棄物)|||kadha, kachara||sampah|takataka|çöp|rubo|roska|śmieć, odpadek
```

Comment:
me volu uza ali loge a me no eureka boni.

Reply:
si, me hema samo. ablo "fey" cha chini? a aga "fey" mena "trash", mome nida muta "fly" pa "vola" o koy shey.

Reply:
mome uza "vol-" pa "to want".